services/rest/.gitignore

```diff
@@ -0,0 +1 @@
+swagger.json
```

Comment:
let's move this up a level to services because I believe it actually show up in webapp (and add api.db)